### DISCUSSION TITLE
Recommendations/feedback on stepper and wizard flow

### DISCUSSION THREAD

**Discussion by ang-twilio** [twilio-labs/paste#1246] [Office hours] [answered]
We are modifying the Messaging Services setup process with a wizard flow to help guide customers through the process of setting up a Messaging Service correctly.

I'm looking for feedback on the Messaging services setup flow specifically:
1. Am I using a step tracker correctly here? Any feedback on what the functionality of a step tracker should include? Ex: should a customer be able to click on a previous step in the step tracker and navigate back to that step?
2. Thoughts on a wizard-like flow using pages instead of modals/dialogs? I think this is appropriate because my flow is pretty complex and can't be contained in a modal. I'm also using modals at times to do something else in the wizard step.

Messaging Services setup flow starting here: https://twilio.invisionapp.com/share/EG10BB7UW68Z#/445823616_step_1_-_MS

**Comment by nkrantz** [comment]
Hey @ang-twilio, thanks for posting your discussion! We'll review and go over this in office hours today.

**Comment by SiTaggart** [comment, marked as answer]
Posting the link to the recoding of the Office Hours https://www.loom.com/share/8b6a519fd2a344dfb94dc966e59a18e6 (password is in [#help-design-system](https://twilio.slack.com/archives/CBEQHHDLP/p1614899487078400) slack)

Lots' of discussion around sizing and placement of large pages and forms. 

- We discovered that most folks don't realise that inputs are 100% and using the standard symbol size
- That the page layout pattern is up coming and needs to consider guidance on max widths of different types of pages

@andoyle did provide some nice context in follow up that new Console pages, according to Jo will 

> “The new Console has 20px on the left, top, and bottom, and 60px on the right”
